.github/workflows/pr-opened.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: PR opened
+
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  add-welcome-message:
```

Comment:
Nice job and great idea. I have quickly checked the action you introduced. The action looks nice and filling a need. But any action we add is always a risk as dependency. Both for maintenance and security. What I don't like on the action there is no provenance of the release. No release automation, and no safe guard in the workflows to ensure the generated file in the dir dist (javascript) is based on the source code in the repo. Since we grant the action write access (only PR), we should always be carefull. 

The welcome message we can also do simply with the github script action. This one is maintained at least by GitHub. See https://github.com/actions/github-script#welcome-a-first-time-contributor

Looking forward to you opinion.

Reply:
Fully agree. Could you please give an example of 

> safe guard in the workflows to ensure the generated file in the dir dist (javascript) is based on the source code in the repo
?